scio-bigquery/src/it/scala/com/spotify/scio/bigquery/types/BigQueryTypeIT.scala

```diff
@@ -47,7 +49,7 @@ class BigQueryTypeIT extends FlatSpec with Matchers {
     "SELECT word, word_count FROM `bigquery-public-data.samples.shakespeare` WHERE word = 'Romeo'"
 
   "fromQuery" should "work with legacy syntax" in {
-    val bqt= BigQueryType[LegacyT]
+    val bqt= BigQueryType[BigQueryTypeIT.LegacyT]
```

Comment:
do a `import BigQueryTypeIT._` at the beginning of the class to avoid these lengthy names?